jbang-catalog.json

```diff
@@ -12,15 +12,15 @@
     "vertx": {
```

Comment:
we really need to move examples out to jbang-examples but thats for a separate issue.